Failed to get last Google Repository package

I have this issue on jenkins when try to build application with GLS dependencie

```
02:29:37.479 [QUIET] [system.out] Checking the license for package Google Repository in /opt/android-sdk-linux/licenses
02:29:37.485 [QUIET] [system.out] License for package Google Repository accepted.
02:29:37.501 [QUIET] [system.out] Preparing "Install Google Repository (revision: 51)".
02:29:37.507 [QUIET] [system.out] Warning: Failed to read or create install properties file.
02:29:37.523 [ERROR] [org.gradle.internal.buildevents.BuildExceptionReporter] 
02:29:37.523 [ERROR] [org.gradle.internal.buildevents.BuildExceptionReporter] FAILURE: Build failed with an exception.
02:29:37.523 [ERROR] [org.gradle.internal.buildevents.BuildExceptionReporter] 
02:29:37.523 [ERROR] [org.gradle.internal.buildevents.BuildExceptionReporter] * What went wrong:
02:29:37.523 [ERROR] [org.gradle.internal.buildevents.BuildExceptionReporter] A problem occurred configuring project ':app'.
02:29:37.523 [ERROR] [org.gradle.internal.buildevents.BuildExceptionReporter] > Failed to install the following SDK components:
02:29:37.523 [ERROR] [org.gradle.internal.buildevents.BuildExceptionReporter]   [Google Repository]
02:29:37.523 [ERROR] [org.gradle.internal.buildevents.BuildExceptionReporter]   The SDK directory (/opt/android-sdk-linux) is not writeable,
02:29:37.523 [ERROR] [org.gradle.internal.buildevents.BuildExceptionReporter]   please update the directory permissions.
02:29:37.524 [ERROR] [org.gradle.internal.buildevents.BuildExceptionReporter] 
02:29:37.524 [ERROR] [org.gradle.internal.buildevents.BuildExceptionReporter] * Try:
02:29:37.524 [ERROR] [org.gradle.internal.buildevents.BuildExceptionReporter] Run with --stacktrace option to get the stack trace. 
02:29:37.524 [LIFECYCLE] [org.gradle.internal.buildevents.BuildResultLogger] 
02:29:37.524 [LIFECYCLE] [org.gradle.internal.buildevents.BuildResultLogger] BUILD FAILED
02:29:37.524 [LIFECYCLE] [org.gradle.internal.buildevents.BuildResultLogger] 
02:29:37.524 [LIFECYCLE] [org.gradle.internal.buildevents.BuildResultLogger] Total time: 10.492 secs
```

--- a/Dockerfile
+++ b/Dockerfile
@@ -22,7 +22,8 @@ ENV ANDROID_HOME /opt/android-sdk-linux
 RUN cd /opt \
     && wget -q https://dl.google.com/android/repository/tools_r$ANDROID_TOOLS-linux.zip -O android-sdk-tools.zip \
     && unzip -q android-sdk-tools.zip -d ${ANDROID_HOME} \
-    && rm -f android-sdk-tools.zip
+    && rm -f android-sdk-tools.zip \
+    && chmod -R 777 ${ANDROID_HOME}
 
 ENV PATH ${PATH}:${ANDROID_HOME}/tools:${ANDROID_HOME}/tools/bin:${ANDROID_HOME}/platform-tools
 

